api/RTCRtpEncodingParameters.json

```
@@ -209,19 +209,19 @@
       },
       "priority": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpEncodingParameters#priority",
+          "spec_url": "https://www.w3.org/TR/webrtc-priority/#dom-rtcrtpencodingparameters-priority",
           "support": {
             "chrome": {
-              "version_added": "67",
-              "notes": "The standard version of this property is <a href='https://developer.mozilla.org/docs/Web/API/RTCRtpSendParameters/priority'><code>RTCRtpSendParameters.priority</code></a>."
+              "version_added": "67"
             },
             "chrome_android": "mirror",
             "edge": {
-              "version_added": "≤79",
-              "notes": "The standard version of this property is <a href='https://developer.mozilla.org/docs/Web/API/RTCRtpSendParameters/priority'><code>RTCRtpSendParameters.priority</code></a>."
+              "version_added": "≤79"
             },
             "firefox": {
               "version_added": "46",
-              "notes": "The standard version of this property is <a href='https://developer.mozilla.org/docs/Web/API/RTCRtpSendParameters/priority'><code>RTCRtpSendParameters.priority</code></a>."
+              "notes": "Version 110 sets to spec default priority to <code>low</code>."
```

What was the previous default priority? I guess 46-110 has a different default priority than the modern spec and this has been fixed?

---

Firefox assigned no default priority at all prior to 110. I don't know what the spec had previously, which is why I just added this as a note to FF. My thinking is that unless we end up with data on other browsers then in "not too long" everyone will use this default. 

But if you think this warrants further investigation, let me know.

---

Looks fine to me. I had trouble understanding the sentence, so I tried to rewrite it:
```suggestion
              "notes": "From version 110 on the default priority is <code>low</code>."
```

---

Thanks! Minor variation on your change done
```suggestion
              "notes": "In version 110 and later the default priority is <code>low</code>."
```